tests/run/cpp_class_attrib.srctree

```diff
@@ -0,0 +1,80 @@
+# tag: cpp
+
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import runner"
+
+######## setup.py ########
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup
+import os
+
+example_dir = os.path.abspath(os.path.join(os.environ['CYTHON_PROJECT_DIR'],
+                              'docs/examples/userguide/wrapping_CPlusPlus'))
+
+ext_modules= cythonize("rect.pyx", include_path=[example_dir])
+setup(ext_modules=ext_modules)
+
+######## rect.pyx ########
```

Comment:
Seems to be this file: https://github.com/cython/cython/blob/master/docs/examples/userguide/wrapping_CPlusPlus/rect_with_attributes.pyx

We shouldn't duplicate it here but compile and test the actual code from the docs.